compiler/rustc_const_eval/src/interpret/eval_context.rs

```diff
@@ -705,13 +710,19 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
         let mut locals = IndexVec::from_elem(dummy, &body.local_decls);
```

Comment:
This line is ~60% total allocated memory for `ctfe-stress-5`.

At first, i tried to cache it too, but it saved later at `self.frame_mut().locals = locals;`, so i don't see easy way to do that.

Reply:
> This line is ~60% total allocated memory for ctfe-stress-5.

Yeah that test involves a lot of CTFE stack variables.

But I don't think this is worth optimizing for. This is totally unrealistic for real code. We shouldn't make rustc harder to maintain just for the sake of meaningless benchmarks -- that is *not* the purpose of these stress tests.